test/model/content_test.dart

```diff
@@ -521,6 +521,16 @@ class ContentExample {
       blockUnimplemented('more text'),
     ]]),
   ]);
+
+  static const thematicBreaks =  ContentExample(
+    'parse thematic break (<hr>) in block context',
+    'a\n---\nb',
+    '<p>a</p><hr><p>b</p>',
```

Comment:
Huh, is this the HTML the server outputs for this input? I'd have guessed there'd be some newlines in there, like between `</p>` and `<hr>`, based on how some other examples in this file look.

Would you link in the PR description to a test message on chat.zulip.org that demonstrates this example? (That's probably something we should start making a practice of, for better reproducibility. Or perhaps even put the link into the source code.)

Reply:
Apologies for the oversight. I adjusted the output for readability purposes, which resulted in a cleaner HTML string. However, I understand the importance of maintaining accuracy for testing purposes. Moving forward, I'll ensure the test output aligns precisely with the server's actual output.

Reply:
Cool, thanks. Yeah, readability is important, but for these tests their job is to make sure we're successfully parsing whatever the server sends us, with all its quirks and sometimes messes — so the top priority is for the data to represent those authentically.

For now let's just add the link in a comment, like this:
```dart
  static const thematicBreaks =  ContentExample(
    'parse thematic break (<hr>) in block context',
    // https://chat.zulip.org/#narrow/stream/7-test-here/near/1774718
    'a\n---\nb',
    '<p>a</p>\n<hr>\n<p>b</p>',
```

Then I'll file a separate issue for filling that in for the existing examples, adding it as a field, and also adding some automated validation of it.

Reply:
Filed #617.